app/components/FlowNavigation.tsx

```diff
@@ -114,39 +106,48 @@ function NavItem({
     stateIsActive(state) &&
     visibleSubflows.some((subflow) => stateIsActive(subflow.state));
   const collapse = useCollapse({ defaultExpanded: isCurrent });
-  const stateClassNames = navItemClassnames(
-    isCurrent,
-    isDisabled,
-    collapse.isExpanded,
+  const stateClassNames = classNames(
+    "p-16 flex gap-x-16 items-center ds-label-02-reg hover:underline",
+    {
+      "ds-label-02-bold hover:no-underline": isCurrent,
+      "text-gray-600 curser-not-allowed hover:font-normal pointer-events-none":
+        isDisabled,
+    },
   );
   const iconId = useId();
 
   return (
     <li
-      key={destination}
-      className={"list-none border-t-2 border-white first:border-t-0"}
+      className={`list-none ${isChild ? "" : "border-t-2 border-white first:border-t-0"}`}
```

Comment:
is it possible to do it with className and negate the value?

```suggestion
      className={`list-none classNames({ "border-t-2 border-white first:border-t-0": !isChild })`}
```

Reply:
see above